database/schema.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE genres (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE authors (
+    id SERIAL PRIMARY KEY,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE labels (
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    color VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE books (
+    id SERIAL PRIMARY KEY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN DEFAULT FALSE,
+    pulblisher VARCHAR(255) NOT NULL,
+    cover_state VARCHAR(255) NOT NULL,
+    genre_id INT REFERENCES genres(id),
+    author_id INT REFERENCES authors(id),
+    label_id INT REFERENCES labels(id)
+);
+
+CREATE TABLE games (
+    id SERIAL PRIMARY KEY,
+    publish_date DATE NOT NULL,
+    archived BOOLEAN DEFAULT FALSE,
+    multiplayer VARCHAR(255) NOT NULL,
+    last_played_at DATE NOT NULL,
+    genre_id INT REFERENCES genres(id),
+    author_id INT REFERENCES authors(id),
+    label_id INT REFERENCES labels(id)
+);
+
+CREATE TABLE albums (
+    id SERIAL PRIMARY KEY,
+    publish_date DATE NOT NULL,
+    on_spotify BOOLEAN DEFAULT FALSE,
+    archived BOOLEAN DEFAULT FALSE,
+    genre_id INT REFERENCES genres(id),
+    author_id INT REFERENCES authors(id),
+    label_id INT REFERENCES labels(id)
+)
```

Comment:
- [OPTIONAL] Kindly create indexes on all foreign keys in this schema based on the requirement. Creating indexes on foreign keys makes queries more efficient. 👍